.github/workflows/test-runner-on-perf.yml

```diff
@@ -62,6 +65,60 @@ jobs:
               fi
             fi
           done
+      - name: "Assemble data-repo code to generate version.properties file"
+        run: ./gradlew clean assemble
+      - name: "Read latest jade-data-repo gitHash from version.properties file"
+        id: read_property
+        uses: christian-draeger/read-properties@1.0.1
```

Comment:
Before merging we should create our own `broadinstitute/read-properties` action for security reasons.

Reply:
Since it was just a couple of lines of code, I just pulled it into our action.